ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -65,6 +65,7 @@
 {% include 'documentation/switchport-port-security.j2' %}
 {# Routing #}
 {% include 'documentation/routing.j2' %}
+{% include 'documentation/ipv6-neighbors.j2' %}
```

Comment:
may be we can remove this one from here if it is added in routing.j2

Reply:
fixed